src/extension.ts

```diff
@@ -1278,16 +1320,42 @@ async function createClusterKubernetes(request? : WizardUIRequest) {
     }
 }
 
-async function useContextKubernetes(explorerNode: explorer.ResourceNode) {
+async function useContextKubernetes(explorerNode: explorer.KubernetesObject) {
     const targetContext = explorerNode.metadata.context;
     const shellResult = await kubectl.invokeAsync(`config use-context ${targetContext}`);
     if (shellResult.code === 0) {
-        vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
+        return vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
```

Comment:
We do this in quite a few places - I would feel more comfortable if we extracted a `refreshExplorer()` method to decouple all these methods from the command ID.

Reply:
`refreshExplorer` needs access a local variable `treeProvider` from `activate()`. Don't find a good way to extract it in the outside like other command handlers. Prefer to keep current design.

Reply:
Sorry I wasn't clear.  The implementation of `refreshExplorer()` would still be `vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");`.  It just prevents repeating the command-based implementation and the command ID in multiple places.

Reply:
Got it. Nice.